index.html

```diff
@@ -21,285 +21,341 @@
-                    <li>Selenium</li>
-                    </ul>
-            </div>
-                <div class="skill">
-               <h3><img src="./images/circleicon.png" class="square-icon" alt="skills-icon">Skills</h3>
-                   <ul>
-                    <li>Codekit</li>
-                   <li>Github</li>
-                   <li>Codepen</li>
-                   <li>Gitlab</li>
-                   <li>Terminal</li>
-                </ul>
-              </div>
-               </div>
-            
+                        <h3><img src="./images/squareicon.png" class="square-icon" alt="square-icon">Frameworks</h3>
+                        <ul>
+                            <li>Bootstrap</li>
+                            <li>Ruby on Rails</li>
+                            <li>RSPec</li>
+                            <li>CapyBara</li>
+                            <li>Selenium</li>
+                        </ul>
+                    </div>
+                    <div class="skill">
+                        <h3><img src="./images/circleicon.png" class="square-icon" alt="skills-icon">Skills</h3>
+                        <ul>
+                            <li>Codekit</li>
+                            <li>Github</li>
+                            <li>Codepen</li>
+                            <li>Gitlab</li>
+                            <li>Terminal</li>
+                        </ul>
+                    </div>
+                </div>
+
             </section>
 
             <section class="form" id="contact">
-                <h3>I'm always interested in hearing about new projects, 
+                <h3>I'm always interested in hearing about new projects,
                     so if you'd like to chat please get in touch.
                 </h3>
 
-                <form action="https://formspree.io/f/xnqlovre" method="POST" >
-                    
-                    <input type="text" name="full-name" placeholder="Full name"   maxlength="30" required> 
-                    
-                   
-                    <input type="email" name="email" placeholder="Email address" required>
-                    
-                    <textarea name="text" placeholder="Enter text here" maxlength="500" cols="37" rows="5" required></textarea>
-                    
-                    <input type="submit" id="sub-btn" value="Get in touch" >
-
-                
+                <form action="https://formspree.io/f/xnqlovre" method="POST">
+
+                    <input type="text" name="full-name" placeholder="Full name" maxlength="30" required>
+
+
+                    <input type="email" name="email" id="email" placeholder="Email address" required>
+
+                    <textarea name="text" placeholder="Enter text here" maxlength="500" cols="37" rows="5"
+                        required></textarea>
+                    <small></small>
+
+                    <input type="submit" id="sub-btn" value="Get in touch">
+
+
```

Comment:
- [ ] [OPTIONAL] It will be nice if you add the error message above/below the submit button 😉 

![image](https://user-images.githubusercontent.com/60164239/132027827-d560f90e-3d2e-41cd-9ba9-0311e3199fcd.png)